components/Footer/index.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+const StyledFooter = styled.footer`
+  position: fixed;
+  padding: 1rem;
+  bottom: 0;
+  background: grey;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Footer = () => {
+  return <StyledFooter />;
+};
+
+export default Footer;
```

Comment:
you don't need another component that only returns your styled component

(be aware that your imports need to be updated as well)

```suggestion
export StyledFooter;
```